### DISCUSSION TITLE
Telemetry exchanged between Components

### DISCUSSION THREAD

**Discussion by pauladam316** [nasa/fprime#1851] [Q&A] [answered]
Hi All,

a question about fprime that I haven't been able to find a good answer on. Can telemetry be exchanged between components, or can it only be downlinked to the GDS? If you want data to be sent both to the ground and to other components, what is the best way to represent it?

**Comment by timcanham** [comment, marked as answer]
Telemetry values as defined by the FPP are meant to go to a GDS that can interpret them using a dictionary. If you wish to share a subset of the values between components, the recommended pattern is to share them via the `Svc/PolyDb` component. You could use a pattern of pushing downlinked telemetry to other components, but you would have to manually decode the binary values back into a usable form, and you also run the risk of getting the IDs and definitions out of sync.
